sql/core/src/test/scala/org/apache/spark/sql/execution/command/DDLSuite.scala

```diff
@@ -1368,7 +1368,7 @@ abstract class DDLSuite extends QueryTest with SQLTestUtils {
       sql("DROP VIEW dbx.tab1")
     }
     assert(
-      e.getMessage.contains("Cannot drop a table with DROP VIEW. Please use DROP TABLE instead"))
+      e.getMessage.contains("dbx.tab1 is a table. 'DROP VIEW' expects a view."))
```

Comment:
@cloud-fan do you think the message is regressing in terms of usefulness? One thing we can do is to introduce `hint: Option[String]` to `UnresolvedView` such that we can make the message as `dbx.tab1 is a table. 'DROP VIEW' expects a view. Please use DROP TABLE instead.`. WDYT?

Reply:
SGTM. Do we need to do the same for table commands?

Reply:
Yes. I will go thru commands that are using `UnresolvedTable` and add hints if necessary as a follow up PR.

Reply:
And I will revisit https://github.com/apache/spark/pull/30475#discussion_r529022520 for `UnresolvedTableOrView`.